interview-app/src/test/java/ua/edu/ukma/interviewapp/InterviewAppApplicationTests.java

```diff
@@ -0,0 +1,14 @@
+package ua.edu.ukma.interviewapp;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class InterviewAppApplicationTests {
+
+    @Test
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.
<!-- __reviewdog__:ChA3ZDdiN2FmMGE3NWM2NmM5EgpjaGVja3N0eWxl -->